rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ClearingRequestSpecTest.java

```diff
@@ -236,4 +232,29 @@ public void should_document_get_clearingrequests_by_state() throws Exception {
                         )));
     }
 
+    @Test
+    public void should_add_comment_to_clearing_request() throws Exception {
+        Map<String, Object> comment = new LinkedHashMap<>();
+        comment.put("text", "comment text 1");
+        comment.put("commentedBy", "test.user@sw60.org");
```

Comment:
`commentedBy` is calculated by the backend using the token. `comment.setCommentedBy(sw360User.getEmail());`

Please do not add it to the documentation. It will cause confusion.